src/theme/shared.ts

```diff
@@ -0,0 +1,66 @@
+interface PaletteOptions {
+  main?: string
+  contrastText?: string
```

Comment:
What is contrastText?
Can you provide more information with a little comment between the types on the interface?